hack/install-gox.sh

```diff
@@ -12,31 +12,18 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.
+# limitations under the License
```

Comment:
That missing period breaks the boilerplate check ;)

Reply:
thanks :smile: